src/schema.ts

```diff
@@ -26,6 +26,8 @@ export namespace CliConst {
     export const CLI_FLATTEN_SET_FLATTEN_PAYLOAD_MAX_POLY_AS_RESOURCE_PROP_KEY = 'cli.flatten.cli-flatten-payload-max-poly-as-resource-prop-count';
     export const CLI_FLATTEN_SET_FLATTEN_PAYLOAD_MAX_POLY_AS_PARAM_PROP_KEY = 'cli.flatten.cli-flatten-payload-max-poly-as-param-prop-count';
     export const CLI_FLATTEN_SET_FLATTEN_ALL_OVERWRITE_SWAGGER_KEY = 'cli.flatten.cli-flatten-all-overwrite-swagger';
+    export const CLI_FLATTEN_SET_FLATTEN_KEEP_UNUSED_FLATTENED_MODELS_KEY = 'cli.flatten.cli-flatten-keep-unused-flattened-models';
+    export const CLI_FLATTEN_SET_FLATTEN_MULTIPLE_REQUEST_PARAMETER_FLATTENING_KEY = 'cli-flatten-multiple-request-parameter-flattening';
```

Comment:
any reason this is not under cli.flatten ?

Reply:
fixed. Thanks